Sources/MapboxDirections/Lane.swift

```diff
@@ -9,29 +9,54 @@ struct Lane: Equatable {
      */
     let indications: LaneIndication
     
+    /**
+     Whether the lane can be taken to complete the maneuver (`true`) or not (`false`)
+     */
     var isValid: Bool
     
-    init(indications: LaneIndication, valid: Bool = false) {
+    /**
+     Whether the lane is a preferred lane (`true`) or not (`false`)
+     A preferred lane is a lane that is recommended if there are multiple lanes available
+     */
+    var isActive: Bool?
+    
+    /**
+     Which of the `indications` is applicable to the current route, when there is more than one
+     */
+    var validIndication: LaneIndication?
+    
+    init(indications: LaneIndication, valid: Bool = false, active: Bool? = false, preferred: LaneIndication? = nil) {
         self.indications = indications
         self.isValid = valid
+        self.isActive = active
+        self.validIndication = preferred
     }
 }
 
 extension Lane: Codable {
     private enum CodingKeys: String, CodingKey {
         case indications
         case valid
+        case active
+        case preferred = "valid_indication"
     }
     
     func encode(to encoder: Encoder) throws {
         var container = encoder.container(keyedBy: CodingKeys.self)
         try container.encode(indications, forKey: .indications)
         try container.encode(isValid, forKey: .valid)
+        try container.encodeIfPresent(isActive, forKey: .active)
+        let str = validIndication?.descriptions.first
+        try container.encodeIfPresent(str, forKey: .preferred)
```

Comment:
Nit: Use a more descriptive constant name or inline it into the next line:

```suggestion
        try container.encodeIfPresent(validIndication?.descriptions.first, forKey: .preferred)
```